scala/friesian.serving/src/main/java/com/intel/analytics/bigdl/friesian/serving/feature/utils/RedisUtils.java

```diff
@@ -76,7 +76,21 @@ public Jedis getRedisClient() {
         return jedis;
     }
 
-    public void Mset(String keyPrefix, List<String>[] dataArray) {
+    public void Mset(String keyPrefix, List<List<String>> dataArray) {
+        if (cluster == null) {
+            jedisMset(keyPrefix, dataArray);
+        } else {
+            clusterSet(keyPrefix, dataArray);
+        }
+    }
+
+    public void Mset(String keyPrefix, String id, String value) {
```

Comment:
You can modify https://github.com/intel-analytics/BigDL/blob/branch-2.0/scala/friesian.serving/src/main/java/com/intel/analytics/bigdl/friesian/serving/feature/utils/RedisUtils.java#L113 setSchema here to set one record.